vms/avm/service_test.go

```diff
@@ -1101,10 +1080,13 @@ func TestServiceGetTxJSON_OperationTxWithMultipleNftxMintOp(t *testing.T) {
 	replyTxBytes, err := json.MarshalIndent(reply.Tx, "", "\t")
 	require.NoError(err)
 
-	expectedReplyTxString := `{
+	sigStr, err := formatting.Encode(formatting.HexNC, mintNFTTx.Creds[1].Credential.(*nftfx.Credential).Sigs[0][:])
+	require.NoError(err)
+
+	expectedReplyTxString := fmt.Sprintf(`{
 	"unsignedTx": {
 		"networkID": 10,
-		"blockchainID": "PLACEHOLDER_BLOCKCHAIN_ID",
+		"blockchainID": "%[1]s",
```

Comment:
(optional, subjective) With such a long format string I'd consider defining the argument slice just before. My hypothesis is that it reduces cognitive load in reading the code because you don't have to repeatedly jump down and back up.

(fyi) Another option for very long blocks like this is to use a `text/template.Template`, which effectively gives you named placeholders instead of argument indices. If you were to do this then I'd abstract a helper function to avoid clogging up the test definitions:

```go
func fmtTemplate(t *testing.T, tmpl string, data any) {
  t.Helper()
  tt, err := template.New("").Parse(tmpl)
  require.NoError(t, err)
  
  var buf bytes.Buffer
  require.NoError(t, tt.Execute(&buf, data))
  return buf.String()
}
...
  expectedReplyTxString := fmtTemplate(t,
    `some long string {{.Foo}} over many {{.Bar}} lines`,
    map[string]any{
      "Foo": "value",
      "Bar": 42,
    } // could also be a struct{Foo string; Bar int}
  )
```

Reply:
The JSON isn't _really_ meant to be read by us. These tests only exist to ensure we don't accidentally change the JSON of any transaction unknowingly.

Moved the argument slice above.